src/index.html

```diff
@@ -2,20 +2,70 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta 
+    <meta
       name="viewport"
       content="
-        width=device-width, 
-        user-scalable=no, 
-        initial-scale=1.0, 
-        maximum-scale=1.0, 
+        width=device-width,
+        user-scalable=no,
+        initial-scale=1.0,
+        maximum-scale=1.0,
         minimum-scale=1.0
       "
     >
     <title>Product cards</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="product-card" data-qa="card">
+      <img
+        src="images/imac.jpeg"
+        alt="produt-image"
+        class="product-card__img"
+      >
+
+      <h2 class="product-card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+
+      <p class="product-card__code">
+        Product code: 195434
+      </p>
+
+      <div class="product-card__review">
+        <div class="stars stars--4">
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+        </div>
+
+        <p class="product-card__review-rate">
+          Reviews: 5
+        </p>
+      </div>
+
+      <div class="product-card__price">
+        <p class="product-card__price-text">
```

Comment:
`p` is not a correct tag here because semantically `p` means a complete piece of information but `Price:` is just a part of a message

So use `p` for entire block but here you need a `span`